### PR TITLE
Move EventLog tests to not run on CI

### PR DESCRIPTION
**Pull request by ViktorHofer**
We are hitting Win32 API issues when calling specific EventLog features.
As the risk of changes in EventLog is low we decided to turn off some
tests in CI and run them manually before a release.

https://github.com/dotnet/runtime/pull/32403#issuecomment-587394723 for discussion.

### CODE REVIEW

**Comment by danmosemsft** [comment]
@Anipik we should add a step as part of shipping 5.0 previews - we should run `IgnoreForCI` tests locally. At the moment, just Windows 10 should be fine. Perhaps this is something you can do after you create each preview branch? 

The goal here is a simple check to make sure we didn't completely break something - EventLog in this case.

**Review by danmoseley — APPROVED** [review]

**Comment by ViktorHofer** [comment]
> @Anipik we should add a step as part of shipping 5.0 previews - we should run IgnoreForCI tests locally. At the moment, just Windows 10 should be fine. Perhaps this is something you can do after you create each preview branch?

Ideally we would just run _all_ tests locally as `IgnoreForCI` tests aren't excluded locally.

**Comment by danmosemsft** [comment]
> Ideally we would just run all tests locally as IgnoreForCI tests aren't excluded locally and it would give us more coverage (local vs CI).

Oh! Of course. We might not get good local coverage of Outerloop, or Linux. But not relevant here: we should be getting coverage for free then. Nice.

File: src/libraries/System.Diagnostics.EventLog/tests/System/Diagnostics/Reader/EventLogRecordTests.cs [hunk 1]
```diff
@@ -29,6 +30,7 @@ public void GetPropertyValues()
         }
 
         [ActiveIssue("https://github.com/dotnet/corefx/issues/34547")]
```
Comment (safern) [inline src/libraries/System.Diagnostics.EventLog/tests/System/Diagnostics/Reader/EventLogRecordTests.cs]: Was this `ActiveIssue` left in purpose?

**Comment by Anipik** [comment]
> @Anipik we should add a step as part of shipping 5.0 previews - we should run IgnoreForCI tests locally. At the moment, just Windows 10 should be fine. Perhaps this is something you can do after you create each preview branch?

I will start doing that for previews

**Pull request merged**